README.md

```diff
@@ -3,10 +3,15 @@
 
 Our goal is to create a survey that helps us, data science & MLOps practitioners, learn more about our community and (in the future) track how our community changes year-over-year.  Said differently - this is a survey for the community, by the community.  We designed this survey to gain an accurate understanding of we, the professionals designing, building and deploying **production** ML systems, individually and collectively are, the types of data and models we implement, the tools we use, and the challenges we face.  Specifically, this is NOT a vendor-led survey that pushes forward a specific perspective that advantages a vendor or group of vendors.
 
-The anonymized responses (e.g., identifying information removed) will be made available as part of this repo.
-
 This document provides an outline of the topics we want to understand from our community.  It is intended to spark discussion and drive alignment on set of topics and learning goals within each topic.
 
+# Objectives & goals
+
+Specifically, our goals are:
+* Develop a ~10 minute survey that provides a neutral, vendor-agnostic view of _who_ is in our community and _why and how_ MLOps is applied.
+* Publish an annual report on the `State of Data Science` based on the survey results
```

Comment:
Should we have it as 'State of MLOps'? (I realise that Data Science is more clickable - so it might be worth using it in a subheading, but is it the main objective?)

Reply:
I think "State of MLOps" is more accurate, but I agree with the click bait attractiveness of "data science"

Let's call is "State of MLOps" internally here since its more accurate, then we can optimize the SEO of the blog seperatly?

Reply:
```suggestion
* Publish an annual report on the `State of MLOps` based on the survey results
```